dashboard/test/ui/step_definitions/steps.rb

```diff
@@ -672,42 +672,6 @@ def generate_user(name)
   }
 end
 
-And(/^I sign in as an admin named "([^"]*)"$/) do |name|
-  steps %Q{
-    Given I am on "http://studio.code.org/reset_session"
-    And I am on "http://studio.code.org/users/sign_in"
-    And I display toast "Loading Rails, creating admin user... (This may take 30 seconds)"
-  }
-
-  require_rails_env
-  email, password = generate_user(name)
-  create_admin_user(name, email, password)
-
-  steps %Q{
-    When I type "#{email}" into "#user_login"
-    And I type "#{password}" into "#user_password"
-    And I click selector "input[type=submit][value='Sign in']"
-    Then I wait to see ".header_user"
-  }
-end
-
-And(/I display toast "([^"]*)"$/) do |message|
```

Comment:
:/ I know this doesn't get used anywhere else (yet) but I'm tempted to say leave it in - it's not specific to being an admin, and it's a nice way to call out particular steps in UI test recordings.